products/mariadb.md

```diff
@@ -45,21 +46,30 @@ auto:
       -   ^mariadb-(?P<major>10)\.(?P<minor>10)\.(?P<patch>([2-9]|\d{2}))$
       -   ^mariadb-(?P<major>10)\.(?P<minor>11)\.(?P<patch>([2-9]|\d{2}))$
       -   ^mariadb-(?P<major>11)\.(?P<minor>[0-7])\.(?P<patch>([2-9]|\d{2}))$
+  -   release_table: https://mariadb.org/about/#maintenance-policy
+      selector: "table"
+      header_selector: "tbody tr:nth-of-type(1)"
+      ignore_empty_releases: true # Ignore future releases
+      fields:
+        releaseCycle: "Release"
+        releaseDate: "GA release date"
+        eol: "Community"
+        eoes: "Extended"
   -   release_table: https://mariadb.org/about/#maintenance-policy
       selector: "table"
       header_selector: "tr:nth-of-type(1)"
       fields:
-        releaseCycle: "Release series"
-        releaseDate: "Stable (GA) Date"
-        eol: "End-of-life"
+        releaseCycle: "Release"
+        releaseDate: "GA release date"
+        eol: "Community"
 
-# Extend/update regex in section above
+# When adding new Major, do no forget to review regexes in section above.
 releases:
 -   releaseCycle: "11.7"
-    releaseDate: 2025-02-13
-    eol: false
+    releaseDate: 2025-02-12
+    eol: 2025-05-12
     latest: "11.7.2"
-    latestReleaseDate: 2025-02-13
+    latestReleaseDate: 2025-02-12
```

Comment:
```suggestion
    latestReleaseDate: 2025-02-13
```
webpage : https://mariadb.com/kb/en/mariadb-11-7-2-release-notes/
![image](https://github.com/user-attachments/assets/967c3c8d-6811-45af-8b0c-27f53dc286b0)
even in github source :
![image](https://github.com/user-attachments/assets/b18eb3f4-e5e6-4792-b436-4abfa85ee320)